guides/development-workflow.md

```diff
@@ -346,23 +318,27 @@ additions.
 
 ## Committing
```

Comment:
I would love some attention on this section in particular.

Reply:
I've included some words to help us think about what should go into commits.

I think step-by-step instructions for _how_ to perform tasks might be too much for this document. It's difficult to know how far to go with instructions (and when to stop) and as you say they're only applicable for git CLI users, you'd need different instructions for GUI users.